### PR TITLE
Add nuspec

### PR DESCRIPTION
**Pull request by jbagga**
nuspec file

### CODE REVIEW

**Comment by natemcmaster** [comment]
How does this work when installed into a project? Is the file copied into build output automatically?

**Comment by Eilon** [comment]
@natemcmaster this package is specifically for non-ASP.NET Core projects, so Content files should work OK.

File: Microsoft.jQuery.Unobtrusive.Ajax.nuspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.jQuery.Unobtrusive.Ajax</id>
+    <version>3.2.4</version>
+    <title>Microsoft jQuery Unobtrusive Ajax</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,aspnet</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
```
Comment (Eilon) [inline Microsoft.jQuery.Unobtrusive.Ajax.nuspec]: Use HTTPS (and verify link works).

File: Microsoft.jQuery.Unobtrusive.Ajax.nuspec [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <tags>jQuery Unobtrusive Ajax</tags>
+    <dependencies>
+      <dependency id="jQuery" version="1.8.0" />
+    </dependencies>
+  </metadata>
+    <files>
+    <file src="dist\jquery.unobtrusive-ajax.js" target="Content"/>
+    <file src="dist\jquery.unobtrusive-ajax.min.js" target="Content"/>
+    <file src="LICENSE.txt" target=""/>
+    <file src="README.md" target=""/>
```
Comment (Eilon) [inline Microsoft.jQuery.Unobtrusive.Ajax.nuspec]: Is this needed?

File: Microsoft.jQuery.Unobtrusive.Ajax.nuspec [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>jQuery Unobtrusive Ajax</tags>
+    <dependencies>
+      <dependency id="jQuery" version="1.8.0" />
+    </dependencies>
+  </metadata>
+    <files>
+    <file src="dist\jquery.unobtrusive-ajax.js" target="Content"/>
+    <file src="dist\jquery.unobtrusive-ajax.min.js" target="Content"/>
+    <file src="LICENSE.txt" target=""/>
```
Comment (Eilon) [inline Microsoft.jQuery.Unobtrusive.Ajax.nuspec]: I don't think this is needed... there's already a license URL.

File: Microsoft.jQuery.Unobtrusive.Ajax.nuspec [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  <metadata>
+    <id>Microsoft.jQuery.Unobtrusive.Ajax</id>
+    <version>3.2.4</version>
+    <title>Microsoft jQuery Unobtrusive Ajax</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,aspnet</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
+    <projectUrl>https://www.asp.net/</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
```
Comment (Eilon) [inline Microsoft.jQuery.Unobtrusive.Ajax.nuspec]: HTTPS

File: Microsoft.jQuery.Unobtrusive.Ajax.nuspec (hunk 2 above)
Comment (jbagga) [reply Microsoft.jQuery.Unobtrusive.Ajax.nuspec]: It's not. Wasn't sure if we wanted it as part of the NuGet package

**Comment by jbagga** [comment]
@Eilon Updated

**Comment by natemcmaster** [comment]
I was just asking because I wasn't sure if the new project system supported the "Content" folder convention in the nuspec. "Content" is a NuGet 2-ism. In NuGet 3, the "contentFiles" feature was added which gives more control to how content items are handled (Compile/Content/Embed), but requires using the `<contentFiles>` element under `<metadata>`.

But if this works, :shipit:.

**Comment by Eilon** [comment]
@natemcmaster yeah I think that is the scenario; it's not meant to work in .NET Core projects.

@jbagga - please confirm that you've tested this in an MVC 5.x project.

**Comment by jbagga** [comment]
Adding them to target="Content/Scripts" correctly places them in the Scripts directory of an MVC 5 app so I am changing the target for both

**Comment by Eilon** [comment]
Ultimately I think we should do whatever the current packages on NuGet.org do. Not sure what that is, but it should be easy to crack open their NUSPECs and see what to do.

**Comment by jbagga** [comment]
Yes! This is what the existing packages on NuGet do as well

**Comment by jbagga** [comment]
@Eilon Added build file. Removing bower.json from here and aspnet/jquery-validation-unobtrusive

**Pull request merged**